Resources/Prototypes/_LostParadise/Entities/Objects/Weapons/Guns/Rifles/M41-A.yml

```
@@ -1,47 +1,47 @@
-        whitelist:
-          tags:
-            - CartridgeLightRifle
-  - type: ContainerContainer
-    containers:
-      gun_magazine: !type:ContainerSlot
-      gun_chamber: !type:ContainerSlot
-  - type: MagazineVisuals
-    magState: mag
-    steps: 1
-    zeroVisible: true
+- type: entity
```

yamllint

[error] wrong new line character: expected \n

(new-lines)